src/ci/docker/wasm32-unknown/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  g++ \
+  make \
+  file \
+  curl \
+  ca-certificates \
+  python \
+  git \
+  cmake \
+  sudo \
+  gdb \
+  xz-utils
+
+RUN curl -sL https://nodejs.org/dist/v9.2.0/node-v9.2.0-linux-x64.tar.xz | \
+    tar -xJ
+
+COPY scripts/sccache.sh /scripts/
+RUN sh /scripts/sccache.sh
+
+ENV TARGETS=wasm32-unknown-unknown
+
+ENV RUST_CONFIGURE_ARGS \
+  --target=$TARGETS \
+  --set build.nodejs=/node-v9.2.0-linux-x64/bin/node
+
+ENV SCRIPT python2.7 /checkout/x.py test --target $TARGETS \
+  src/test/ui \
+  src/test/run-pass \
+  src/test/compile-fail \
+  src/test/parse-fail \
+  src/test/run-fail \
```

Comment:
I think it's better to remove this line over opting-out in `bootstrap/check.rs`? There needs to be a FIXME comment next to one of the workarounds anyway.

Reply:
Sure yeah, I preemptively added that to get a `./x.py test --target wasm32-...` working, but we can always easily add it back later.